partner_industry_parent/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Partner Industry Parent',
+    'summary': """
+        This module add a parent relation to the partner industry""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,' 'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': ["base"],
+    'data': ["views/res_partner_industry.xml"],
+    'demo': [],
```

Comment:
@sbejaoui no need then remove it

Reply:
@nikul-serpentcs , look [here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#modules).

>  The `website` key must be `https://github.com/OCA/<repo>`,
>     so as to provide the most relevant link to discover more information about the addon.
>     That link shows the repository README. Alternatively `https://github.com/OCA/<repo>/tree/<branch>/<addon>`
>     may be used, to provide a direct link to the addon README, which includes proper credits
>     (authors, contributors and theirs companies), and links to the relevant information on the OCA website.

Reply:
@sbejaoui Yes, If not any file then remove it, no need.

It's not a big deal!! :smile: